tests/test_qubit_layout.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import os
+import unittest
+
+import dwave_networkx as dnx
+
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    plt = False
+
+try:
+    import numpy as np
+except ImportError:
+    np = False
+
+_display = os.environ.get('DISPLAY', '') != ''
+
+
+@unittest.skipUnless(np and plt, "No numpy or matplotlib")
+class TestDrawing(unittest.TestCase):
+    @unittest.skipUnless(_display, " No display found")
```

Comment:
Not sure you need this if using non-interactive matplotlib backend. Btw, which backend is used here? Hopefully not interactive..

Reply:
I added this for consistency with existing tests, e.g., https://github.com/dwavesystems/dwave-networkx/blob/main/tests/test_pegasus_layout.py#L68-L104